src/woo_publications/api/models.py

```diff
@@ -0,0 +1,76 @@
+import binascii
+import os
+
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from django_jsonform.models.fields import ArrayField
+from phonenumber_field.modelfields import PhoneNumberField
+
+from .constants import PermissionOptions
+
+PERMISSION_SCHEMA = {
+    "type": "list",
+    "items": {
+        "type": "string",
+        "choices": [
+            {"title": label, "value": value}
+            for value, label in PermissionOptions.choices
+        ],
+        "widget": "multiselect",
```

Comment:
```suggestion
```
not needed - my sample code excluded it so that the UI made it clear you're dealing with a list of permissions.

![image](https://github.com/user-attachments/assets/39c41a37-c116-40e6-8d44-774e262cfdeb)

Does not look great tbh